### ISSUE TITLE
Custom Policy Network DQN [ LaneEnv ]

### ISSUE THREAD

**Issue by alt113** [traffic-control-fyp-aub/ns3-gym#43]
Train Custom Policy Network DQN algorithm on LaneEnv with Zoo Hyperparams:

- **Number of input neurons:** 13
- **Number of hidden layers:** 2
- **Number of neurons per hidden layer:** 100
- **Connections between layers:** fully connected
- **Number of output neurons:** 3
- **Activation function:** ELU
- **Mini-batch size:** 5
- **Weights initialization:** 1 / sqrt(num_hidden_nodes)
- **Bias value initialization:** 0.01
- **Gamma:** 0.99
- **Learning rate for Adam:** 1e-5
- **Number of training iterations:** ~100
